project/maintainers.md

```diff
@@ -0,0 +1,237 @@
+dealing with contributors.
+
+Failure to be fair or impartial reflects poorly on the released product, as
+guilt by association taints the product.  The process of reviewing a merge
+request often includes conflict.  Contributors can become defensive about work
+they've done while maintainers can become adamant in the changes they request.
+
+To prevent a breakdown in the review process, the project encourages all
+reviewers to adhere to a standard set of review best practices.  Reviewers
+should familiarize themselves with these practices and suggest updates to keep
+the practices relevant over time.
+
+## Review Standards
+
+These standards serve to prevent problems from cropping up during a review.  The 
+intent is that consistent application of these standards permits a consistent 
+review process, leading to repeatable, suprise free, outcomes during a review.
+
+The intent of maintaining standards is to enhance productivity and improve team
+morale.  In the event the standards have a negative impact on productivity or
+morale, the standard itself should be questioned.  To clarify the kinds of
+productivity to be improved, the intent is to reduce the time between initial
+submission of a merge request and its resolution.
+
+### Challenges to the Review Process
+
+Whenever possible, a maintainer should not argue a point about the review
+standards with a contributor.  Instead they should provide this document to the
+contributor, indicating that changes to the review process are to be initiated
+with a standard-altering Issue.
+
+In the exceedingly rare situation that a reviewer opts to ignore a review
+standard during a merge request, the reviewer must indicate they are purposefully
+ignoring the standard and the reason why.  There are valid reasons to ignore
+standards, but whenever possible a maintainer should uphold the standard or
+change it.
+
+### Review Outcomes
+
+From the Maintainers point of view, all merge requests require one of four
+actions:
+
+- The maintainer accepts the pull request.
+- The maintainer rejects the pull request.
+- The maintainer requests the pull request be altered.
+- The maintainer rewrites some or all of the pull request, becoming a
+  committer.
+
+Suggestions to the committer that a merge request be altered do not constitute
+becoming an committer, even if the maintainer provides the lines of code
+being suggested.
+
+### Review Tempo
+
+Maintainers should set aside and appropriate amount of time when reviewing.  The
+initial suggestion is one hour.  Most reviews will complete well under this time,
+but a few will take longer.  Longer reviews should include review breaks, so the
+reviewer remains fresh and attentive.  Attempting a three hour long review often
+yields worse results than two or three shorter efforts with breaks.
+
+The concept of going slow to complete items quickly is not a new one.  With a
+little extra time, comments can be thoughtful instead of reactive.
```

Comment:
```suggestion
Maintainers should set aside and appropriate amount of time when reviewing. The
initial suggestion is one hour. Most reviews will complete well under this time,
but a few will take longer. Longer reviews should include review breaks, so the
reviewer remains fresh and attentive. Attempting a three hour long review often
yields worse results than two or three shorter efforts with breaks.

The concept of going slow to complete items quickly is not a new one. With a
little extra time, comments can be thoughtful instead of reactive.
```